src/java.base/share/classes/java/lang/foreign/ValueLayout.java

```diff
@@ -465,7 +513,7 @@ sealed interface OfDouble extends ValueLayout permits ValueLayouts.OfDoubleImpl
     OfChar JAVA_CHAR_UNALIGNED = JAVA_CHAR.withBitAlignment(8);
 
     /**
-     * An unaligned value layout constant whose size is the same as that of a Java {@code short}
+     * A nameless unaligned value layout constant whose size is the same as that of a Java {@code short}
```

Comment:
I would avoid saying "nameless" here - names are optional - if we don't say anything, it implies that users shouldn't make any assumption on the name being there.